src/Sentry/Internal/Http/CachingTransport.cs

```diff
@@ -187,6 +187,12 @@ private async Task ProcessCacheAsync(CancellationToken cancellationToken = defau
                 }
                 catch (Exception ex) when (IsRetryable(ex))
                 {
+                    _options.DiagnosticLogger?.LogError(
+                        "Failed to send cached envelope: {0}, retrying after a delay.",
```

Comment:
Do we just retry after a delay? At this time only on restart on next time an event is sent right?
```suggestion
                        "Failed to send cached envelope: {0}. It will be retired later.",
```

Reply:
Tried to be consistent with the other one

![image](https://user-images.githubusercontent.com/1935960/107666900-b5bf3280-6c97-11eb-96f2-a7834b2e9202.png)

Reply:
I don't follow. Maybe the other messages are misleading?

Just poining out that `retrying after a delay` isn't true